app/views/layouts/_navigation.html.haml

```diff
@@ -13,17 +13,32 @@
         %ul.nav.navbar-nav#splash-nav
           = content_for :splash_nav
       -if user_signed_in?
-        %ul.nav.navbar-nav.navbar-right
-          %li.dropdown
-            %a.dropdown-toggle{"data-toggle" => "dropdown", :href => "#", id: "current-user-detail"}
-              - if not current_user.name.blank?
-                #{current_user.name}
-              -else
-                #{current_user.email}
-              = image_tag(current_user.gravatar_url(size: '18'), title: "Yo #{current_user.name}!", :alt => '')
-              %b.caret
-            %ul.dropdown-menu
-              = render 'layouts/user_menu'
+        .btn-group.pull-right
+          %ul.nav.navbar-nav.navbar-right
+            %li.dropdown
+              %a.dropdown-toggle{"data-toggle" => "dropdown", :href => "#", id: "current-user-detail"}
+                - if not current_user.name.blank?
+                  #{current_user.name}
+                -else
+                  #{current_user.email}
+                = image_tag(current_user.gravatar_url(size: '18'), title: "Yo #{current_user.name}!", :alt => '')
+                %b.caret
+              %ul.dropdown-menu
+                = render 'layouts/user_menu'
+          - if can? :manage, Conference
```

Comment:
Yes, there is a cfp role too.

Based on the changes in models/ability.rb we want users with the role **organizer** and the role **cfp** to be able to manage comments. Is that right @raluka ?

"if can? :manage, Conference" means *if the user can manage some conference (any conference)*. That makes sense to cover the users with the role **organizer** but not the ones with the **cfp** role.

Moreover roles' abilities may change eventually, and this statement depends on the roles that the organizer or a cfp team member has. But what we really care about here is if the user can manage comments, right @raluka ? (regardless of the user's role or ability to manage or not manage a conference)
So I would suggest to check about that:
- if can? :manage, Comment

I also have concerns about the **manage** ability.
Isn't this reallly about being able to **show** the comments? Just because you get the notification and you can see the comment, it doesn't necessarily mean you can delete the comment or edit the comment or even create a new comment. But this is something we can work on afterworks too, as it doesn't directly affect any operation in OSEM.